code/modules/projectiles/guns/energy/hos.dm

```diff
@@ -0,0 +1,273 @@
+// -------------- HoS Modular Weapon System -------------
+// ---------- Code originally from VoreStation ----------
+/obj/item/gun/ballistic/revolver/mws
```

Comment:
This is a revolver, move it to `code/modules/projectiles/guns/ballistic/revolver.dm`